contractcourt/channel_arbitrator_test.go

```diff
@@ -686,13 +704,95 @@ func TestChannelArbitratorBreachClose(t *testing.T) {
 	// It should start out in the default state.
 	chanArbCtx.AssertState(StateDefault)
 
+	outgoingIdx := uint64(2)
+
+	rHash1 := [lntypes.PreimageSize]byte{1, 2, 3}
```

Comment:
nit: can we add some docs here saying we've created one outgoing and one incoming HTLCs on the remote commit and test that only the outgoing one is resolved? Can be helpful for future developers.

Reply:
Added